core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.ts

```diff
@@ -130,10 +131,12 @@ oppia.directive('audioTranslationBar', [
           $scope.unsavedAudioIsPlaying = false;
           $scope.waveSurfer = null;
           document.body.onkeyup = function(e) {
-            if (e.keyCode === 82 && !$scope.isAudioAvailable) {
-              // 82 belongs to the keycode for 'R'
-              // Used as shortcut key for recording
-              toggleStartAndStopRecording();
+            if (RouterService.getActiveTabName() === 'translation') {
```

Comment:
I think this would resolve the issue, can you check whether we can remove this listener entirely if someone leaves the translation tab?

Maybe `$scope.$on('$destroy', ..` will be useful.

Reply:
What do you mean by removing this listener? 
Something like that `document.body.onkeyup = undefined` ?

Reply:
If it removes the event listener then I'm fine.